### PR TITLE
Facebook auth

### PR DESCRIPTION
**Pull request by matthewglover**

### CODE REVIEW

File: server/routes/login_routes/welcome.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+      callback(null, response.body);
+    });
+};
+
+const welcome = {
+  method: 'GET',
+  path: '/welcome',
+  handler(request, reply) {
+    getAccessToken(request.query.code, (accessTokenError, accessToken) => {
+      if (accessTokenError) throw accessTokenError;
```
Comment (daymos) [inline server/routes/login_routes/welcome.js]: you are throwing error here, I think for production we should handle it differently

File: server/routes/login_routes/welcome.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      getUserDetails(accessToken, (userDetailsError, userDetails) => {
+        if (userDetailsError) throw userDetailsError;
```
Comment (daymos) [inline server/routes/login_routes/welcome.js]: same here

**Comment by daymos** [comment]
looks good to me. I made one comment in regard to handling errors. Let me know what you think

File: server/routes/login_routes/welcome.js (hunk 1 above)
Comment (gabrielperales) [inline server/routes/login_routes/welcome.js]: yes, maybe we should raise an  401 or 403, we were talking about handle it differently, but ad the end we left it like that.

**Pull request merged**